programs/clearing_house/src/state/user.rs

```diff
@@ -317,11 +316,9 @@ pub struct Order {
     pub reduce_only: bool,
     pub post_only: bool,
     pub immediate_or_cancel: bool,
-    pub discount_tier: OrderDiscountTier,
```

Comment:
nice, so the user stats will infer this

Reply:
im thinking if we wanted to do something like this, we would probably want to force user to stake the discount token and store the amount staked on the user stats account